conf.d/abbr_tips.fish

```diff
@@ -33,14 +33,25 @@ function __abbr_tips --on-event fish_postexec -d "Abbreviation reminder for the
         end
     else if test "$command[1]" = "alias"
         # Update abbreviations list when adding aliases
-        and not contains -- "$command[2]" $__ABBR_TIPS_KEYS
-        if test (count $command) = 2
+        set -l alias_key
+        set -l alias_value
+
+        if string match -q '*=*' "$command[2]"
             set command_split (string split = $command[2])
-            set -a __ABBR_TIPS_KEYS $command_split[1]
-            set -a __ABBR_TIPS_VALUES $command_split[2]
+            set alias_key $command_split[1]
+            set alias_value $command_split[2]
+            set -a alias_value $command[3..-1]
+        else
+            set alias_key $command[2]
+            set alias_value $command[3..-1]
+        end
+
+        if set -l abb (contains -i -- "$command[3]" $__ABBR_TIPS_KEYS)
+            set __ABBR_TIPS_KEYS[$abb] $alias_key
+            set __ABBR_TIPS_VALUES[$abb] (string trim -c '\'"' $alias_value | string join ' ')
         else
-            set -a __ABBR_TIPS_KEYS $command[2]
-            set -a __ABBR_TIPS_VALUES $command[3]
+            set -a __ABBR_TIPS_KEYS $alias_key
+            set -a __ABBR_TIPS_VALUES (string trim -c '\'"' $alias_value | string join ' ')
```

Comment:
My previous exemple (`alias gq 'grep -q'`) still not works because `-q` is interpreted as a `string` option. We should use the `--` argument here (`string trim -c '\'"' -- $alias_value | string join ' '`) to ensure that the content of `alias_value` is interpreted as an argument.

Reply:
Ah! I’d actually made this change but forgot to commit it. I’ll update. :)